src/client/common/installer/productInstaller.ts

```diff
@@ -277,8 +280,8 @@ export class ProductInstaller implements IInstaller {
     ): Promise<InstallerResponse> {
         return this.createInstaller().install(product, resource, cancel, reInstallAndUpdate, installPipIfRequired);
     }
-    public async isInstalled(product: Product, resource?: InterpreterUri): Promise<boolean | undefined> {
-        return this.createInstaller().isInstalled(product, resource);
+    public async isInstalled(product: Product, interpreter: PythonEnvironment): Promise<boolean | undefined> {
```

Comment:
This is now cleaner, we were always passing `PyhtonEnvironment`, but was left as `InterpreterUri` as this was a carry over from from Python extension